packages/eth-rpc-cache/src/strategies/per-block.ts

```diff
@@ -35,20 +30,8 @@ const perBlockMethods = [
 
 const methods = [...mayBeSafeMethods, ...perBlockMethods]
 
-const getRpc = (
-  rpc: JsonRpcCallFn,
-  cache: Map<string, unknown>,
-  options = {}
-) =>
-  pMemoize(rpc, {
-    cache,
-    maxAge: 6000, // Half block time: ~6 sec.
-    resolver: (method: string, params: unknown[]) => getKey(method, params),
-    ...options
-  })
-
 export const perBlockStrategy: Strategy = {
-  getRpc,
+  maxAge: 6000, // Half block time: ~6 sec.
```

Comment:
I wonder if we should convert this into a constructor function, so `maxAge` can be parameterized

```js
const perBlockStrategy = ({ maxAge = 6000 }) => ({
  maxAge,
  methods,
  name: 'block'
})
```

This way it is exensible for chains that use a different block time and are evm compatible (For `permanent` strategy, there would be no parameters, but I would do it as a constructor still for consistency)

Reply:
This override can be done today through the `options` passed to `createEthRpcCache()`. Let's keep thinking on how to better address this problem for a future version.

Reply:
true! We actually discussed about it, I forgot it 😆